.github/workflows/ci.yml

```diff
@@ -129,39 +137,39 @@ jobs:
     name: Build Windows
     uses: ./.github/workflows/build-native.yml
     with:
-     runner: windows
+     runner: windows-latest
      binary: windows
      build: '["windows"]'
 
   build-macos:
     name: Build MacOS
     uses: ./.github/workflows/build-native.yml
     with:
-     runner: macos
+     runner: macos-latest
      binary: macos
      build: '["macos"]'
 
   build-linux:
     name: Build Linux
     uses: ./.github/workflows/build-native.yml
     with:
-     runner: ubuntu
+     runner: ubuntu-20.04
```

Comment:
Should we have a comment here why we use a specific version for the Linux build. Something like

"Building on the lowest possible Linux version for compatibility"